lib/typescript/httpclient/api/airyConfig.ts

```diff
@@ -34,10 +34,16 @@ export const doFetchFromBackend = async (url: string, body?: Object): Promise<an
 
 async function parseBody(response: Response): Promise<any> {
   if (response.ok) {
-    return response.json();
+    try {
+      let jsonResponse = await response.json();
```

Comment:
const should do it?

Reply:
will just do return await response.json()